### ISSUE TITLE
DatePicker: press enter fail to trigger Picker open when disable auto focus with JAWS 17 opened

### ISSUE THREAD

**Issue by MirandaGe** [OfficeDev/office-ui-fabric-react#1790]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: IE 11 with JAWS 17

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: High

Products/sites affected: events web part

#### Describe the issue:

1. add a datePicker with disableAutoFocus=true, Open JAWS
2. let focus goes to the input area of DatePicker, select a date.
3. After select the date, focus is on the textfield of DatePicker with callout not opened. Press enter.
3. The Callout fail to open for the first time. If I press enter again, it will pop up. If JAWS 17 is closed, it can triggered as expected

#### Expected behavior:

The Callout can pop up first time I press enter with JAWS opened.

### If applicable, please provide a codepen repro:

https://codepen.io/lingge/pen/rmreQy
